src/utilities/email/index.ts

```diff
@@ -0,0 +1,5 @@
+export function isValid(email: string): boolean {
+  if (!email || typeof email !== 'string') return false;
+
+  return /^[a-zA-Z0-9_.-]+@[a-zA-Z0-9-]+.[a-zA-Z0-9-.]+$/.test(email);
```

Comment:
In my researches i found that the email validation is a little bit more complex than this.

https://help.returnpath.com/hc/en-us/articles/220560587-What-are-the-rules-for-email-address-syntax-

You can take a look at this library that i used once to validate email the regex is more complex:

https://github.com/manishsaraan/email-validator/blob/master/index.js

Reply:
Nice! Can you help me to implement this?

Reply:
Considering the article you mentioned. This validation is much more complex than the library you used.
For example, in the article or author says that emails should not start with special characters, in the library this result is a valid email.

Reply:
> @hyanmandian 
> Nice! Can you help me to implement this?

Sure!

> @fxamauri 
> Considering the article you mentioned. This validation is much more complex than the library you used.
> For example, in the article or author says that emails should not start with special characters and should not appear consecutive, in the library this result is a valid email.

I will take a look at more articles to make sure will be all done right.

Reply:
Ok. If you need help, I’m at your disposal

Reply:
I was creating some tests following the article and they talk about the `Top level domain`. I found a list with all the top level domains, it can be found [here](https://www.icann.org/resources/pages/tlds-2012-02-25-en). How do you guys think that this implementation should done?  Creating a new helper package, add this list file to the email validation package, we don't need to validate this etc

@fxamauri @hyanmandian

Reply:
I think that's a good idea to create a JSON file inside `is-valid-email` with the domains. We just need to confirm if we really need to validate these domains before.

Reply:
I found that the TLD list is updated weekly and all public TLD has at least 2 chars length and in intranet cases it can has 1 char length. So i think that we should only check if the length is not zero

Reply:
Nice! In that way, we don't need to store TDL list (and update every week)